.gitignore

```diff
@@ -8,3 +8,4 @@ benchmarks
 tags
 .DS_Store
 docs/_site
+gemfiles/*.lock
```

Comment:
Let's remove this and commit the lock file so we have consistency with the other ActiveAdmin organization gems.

Reply:
I've tried but I had issues with `racc` and jruby, but I think I can make it work